### DISCUSSION TITLE
The two nRF platforms

### DISCUSSION THREAD

**Discussion by rahav** [contiki-ng/contiki-ng#2683] [Q&A] [answered]
Which of the ports [nrf52840: Nordic Semiconductor nRF52840 (nRF5 SDK)](https://docs.contiki-ng.org/en/develop/doc/platforms/nrf52840.html) or
[nrf: Nordic Semiconductor nRF5340 and nRF52840 (nRF MDK)](https://docs.contiki-ng.org/en/develop/doc/platforms/nrf.html) is going to prevail after they are merged?

What is the difference between the SDK and MDK?

Thanks in advance, Rahav

**Comment by pjonsson** [comment, marked as answer]
It's the nrf platform that is meant to stay, since that also supports the nrf 5340. Off the top of my head, the missing feature of the nrf platform that is available in the nrf52840 platform is the DFU support.

**Reply by Yagoor** [reply]
Correct. I have added USB support to the nrf platform, however, Nordic does not use the DFU Device Class for DFU they have their own class for it which I didn't implement.

**Reply by rahav** [reply]
Thank you both. Right now I'm using the nrf52840 platform, do I need to take into account changes that will be introduced because of the move from SDK to MDK?
